pyemma/coordinates/data/_base/iterable.py

```diff
@@ -20,16 +20,18 @@
 import numpy as np
 
 from pyemma._base.loggable import Loggable
-from pyemma._base.progress import ProgressReporterMixin
 from pyemma.util.contexts import attribute
 from pyemma.util.types import is_int
 
+# this is used, in case None is passed as input chunk size.
+DEFAULT_CHUNKSIZE = 5000
+
 
 class Iterable(Loggable, metaclass=ABCMeta):
 
-    def __init__(self, chunksize=1000):
+    def __init__(self, chunksize=DEFAULT_CHUNKSIZE):
```

Comment:
i believe this is inconsistent, the default chunk size is set if the argument is `None`, however the argument itself defaults to the default chunksize

Reply:
also it is called `chunksize` instead of `chunk_size`